### ISSUE TITLE
I cannot access entire list of articles when I use the @ feature to tag an article

### ISSUE THREAD

**Issue by avak8848** [worldanvil/worldanvil-bug-tracker#176]
**World Anvil Username**: scribe.constance

**World / Campaign / RPG Character / Manuscript Title**: BOS: Operation hOpe4FUTURE

**Feature**: tagging system

**Related URLS** (To public & visible pages add screenshots otherwise):

**Describe the Issue**
When I use the "@" symbol to link and article within another article, the dropdown list contains only a few articles with the word in it and not all of them

**To Reproduce**

Go Dashboard and create a new article. 
In the main text box type "@" and the name of another article that you want to link within the new one
See that the drop down box only has a few options and that you cannot scroll to see the rest like in the past 

**Expected behavior**
I expected to be able to scroll down in the drop down menu to find every article containing the word I typed

**Comment by iamromeo** [comment]
This is not a bug, the chances are you have more articles with similar name than the ones the suggestion system will return. 
You can use any part of the article's name to get results back. Maybe you can try searching for those parts? 

Is there a reason you wouldn't be able to?

**Comment by avak8848** [comment]
I have recorded a video that shows the problem. I think my bug report
didn't really capture the entire issue. Here is the link
https://youtu.be/7b5x4nCQDFw

**Comment by Kordishal** [comment]
Hi Ava!

When I create an article named scorched and search for it it works no problem. Can you link that article? Are you sure there is an article with that word in the title? We need to better investigate this issue to find the reason it does not bring up any result for that one.

The limitations on the list I believe makes sense so using different words for different searches will always be required if you have many similar titles. On a side note you can search for tags in this list as well. So if you have an article with too generic a name you may want to give it a unique tag that you can search for to bring it up.

**Comment by avak8848** [comment]
https://www.worldanvil.com/w/bos3A-operation-hope4future-head-librarian-carlson/a/scorched-article

Here is the Scorched article. We have tagged it many times in other
articles so it used to work. It also used to bring up the Scorchbeast
article (
https://www.worldanvil.com/w/bos3A-operation-hope4future-head-librarian-carlson/a/scorchbeast-28sierra-bravo29-article)
but that is not showing up either.

**Comment by Kordishal** [comment]
Can you check if the Console has any errors related to the mention system?

Press F12 and then select the Console tab. Then attempt to link scorched.

**Comment by avak8848** [comment]
I attempted tagging Scorched and it didn't let me tag it but there were no
errors in the console.

**Comment by Kordishal** [comment]
What do you mean it didn't let you tag it? Tags can be added in the tags field. This has never been an issue.

![image](https://user-images.githubusercontent.com/14262536/116786287-09bf0380-aa9e-11eb-9fe9-53dbffa38b80.png)
You can add tags in the Articles & Categories sidebar when selecting an article.

**Comment by avak8848** [comment]
Sorry, let me rephrase. It didn't let me use the @ in the body text of the
article to mention scorched. Like, the drop down menu didn't pop up.

**Comment by Kordishal** [comment]
But if you add some tag to the article (can just be the word `test` (something that isn't in your world). Can you then find it via the mention system?

**Comment by avak8848** [comment]
No, I still can't find it after tagging it on the side.

**Comment by Kordishal** [comment]
Ok. Can you make a second article named scorched and test if you can find that one?

**Comment by avak8848** [comment]
I made a second article and that one does not show up in the drop down menu
either.

**Comment by Kordishal** [comment]
Well I am out of ideas what to try. It seems to me for some reason those articles are not in the index for the tagging system. But why a new article would not appear ...

@iamromeo Am unable to reproduce this or figure out what is wrong.

**Comment by ScribeCarlson** [comment]
I am having this same issue on my end with multiple articles on the site.
